com.musala.database.web.spa.parser/WebContent/endToEndTesting/ProtractorSpecification.js

```diff
@@ -0,0 +1,130 @@
+    
+var urlLocation='http://localhost:8080';
+var packageName='/com.musala.database.web.spa.parser';
```

Comment:
Please put an empty space before and after every assignment sign (=). Valid for the whole file.